elastic/kibana
# fix kibana metricsets property paths

### Summary

The metricbeat path for kibana queries currently expects an inner `kibana.stats.kibana` object to contain a summary of the instance similar to the legacy schema, but this object does not exist. Instead the properties making up this summary are spread in the `kibana.stats` object. This change updates codepaths relying on this non-existing object.

Also updated the `_mb` kibana archives (with [this script](https://gist.github.com/klacabane/a180c5764dacdde8d181d018aa759672)) used by the apis and functional tests to 1. align with the actual metricbeat schema 2. insert documents in the new datastreams.

### Testing
- start metricbeat with kibana module enabled
- ensure all Stack monitoring kibana views are loading and displaying relevant data
